layouts/default/60_ansi_arrow/default_60_ansi_arrow/keymap.c

```diff
@@ -15,39 +15,30 @@
  * along with this program. If not, see <https://www.gnu.org/licenses/>.
```

Comment:
Was there a reason to update this default community layout?

Reply:
Just a little updates